CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+Welcome to Stream’s Android repository! We welcome all feedback and we hope this document helps you contribute to our repo! 🎉.
+
+We'll guide you through our repository structure, present our styling guidelines and describe the pull request and approval process.
+
+These are guidelines and don't have to be followed to a letter, if you're looking to contribute, feel free to do your best and what you're comfortable with, and we'll take care of the rest! 🤗
+
+---
+
+# If I have a question, do I need to read this guide? 💬
+
+Not really! This guide is focused on contributions, so if you have questions, want to report a bug or want to request a new feature, use our [Android Chat SDK Task and Bug Report](https://github.com/GetStream/stream-chat-android/issues/new/choose) templates.
```

Comment:
Should we redirect to discussions instead or at least mention them?

Reply:
That's a good point - I'll add the first step to be look into discussions